web/themes/custom/my_awesome_theme/components/03-organisms/site/site-footer/_site-footer.scss

```diff
@@ -16,30 +13,62 @@
   }
 }
 
-.footer__social {
-  margin-bottom: $space;
-
-  @include large {
-    flex: 0 1 30%;
-    margin-bottom: 0;
-    margin-right: $space;
-  }
+.footer:before {
+  background-image: url('https://demo.tagdiv.com/newspaper_pro/wp-content/uploads/2021/06/footer-bg1.jpg');
```

Comment:
Place assets locally plz.

Reply:
@HeorhiiBoliuk This hasn't been fixed yet
You can explain why you think this is better in case you don't want to change it.
Or ask if you don't understand what should be fixed here